### PR TITLE
Update mkdocs.yml

### PR DESCRIPTION
**Pull request by abdullahvenly**
Added two links:
1. NFT-API: Which is a guide on how to create and mint NFTs on the Polygon chain.
2. Zapier Integration: Which is again another guide for low-code users that teaches them how to create and mint NFTs on the Polygon chain.

### CODE REVIEW

**Comment by abdullahvenly** [comment]
Hi @kmurphypolygon 

Added them as external links under `mkdocs.yml` as you suggested.

File: mkdocs.yml [hunk 1]
```diff
@@ -378,6 +378,8 @@ nav:
           - Usage: tools/chain-indexer-framework/usage.md
       - Third-party tutorials:
           - Alchemy subgraphs: https://docs.alchemy.com/docs/how-to-build-and-deploy-a-subgraph-on-polygon-zkevm-using-alchemy-subgraphs
+          - Venly's NFT-API: https://docs.venly.io/docs/nft-api-getting-started
+          - Venly's Zapier Integration: https://docs.venly.io/docs/zapier-integration
```
Comment (kmurphypolygon) [inline mkdocs.yml]:
```suggestion
          - Venly's Zapier integration: https://docs.venly.io/docs/zapier-integration 
```

**Comment by kmurphypolygon** [comment]
> Hi @kmurphypolygon
> 
> Added them as external links under `mkdocs.yml` as you suggested.

Looks good, thank you :)

**Review by kmurphypolygon — APPROVED** [review]

**Comment by abdullahvenly** [comment]
Thanks for approving @kmurphypolygon 

Do you have any other way I can ask you questions about adding/updating documentation? Preferably Slack or if you have any other option? thnks

**Comment by kmurphypolygon** [comment]
> Thanks for approving @kmurphypolygon
> 
> Do you have any other way I can ask you questions about adding/updating documentation? Preferably Slack or if you have any other option? thnks

At the moment there hasn't been a need to support external chat or messaging. What sorts of things were you thinking about?

**Comment by abdullahvenly** [comment]
Well then I'll just ask questions whenever I open a PR. No questions atm. thnks

**Comment by kmurphypolygon** [comment]
> Well then I'll just ask questions whenever I open a PR. No questions atm. thnks

Cool, great, if things start getting busy, we can rethink and I'll set up a slack channel for externals ... who do you liaise with in dev rel at the moment?

**Pull request merged**